src/highlight.rs

```diff
@@ -109,6 +126,13 @@ fn highlight(source: &str, ext: &str) -> Result<String, Error> {
             html.push_str(&"<span>".repeat(delta.abs().try_into()?));
         }
 
+        // Process link element placeholders.
+        formatted = LINK_REGEX_2.replace_all(&formatted, |x: &Captures| {
+            let num: usize = x.get(1).unwrap().as_str().parse().unwrap();
```

Comment:
The second `unwrap()` can be reached, e.g. via paste containing `LINKWASTEBIN99999999999999999999999END`